### DISCUSSION TITLE
I'm not seeing how this works, can you help?

### DISCUSSION THREAD

**Discussion by Larzous** [sarisia/actions-status-discord#528] [Q&A] [answered]
I'm newer to github actions and I'm just not seeing where/how the scripts runs so I can look at the javascript that is actually running.

In `actions.yml` it says:
```js
runs:
  using: "node20"
  main: "lib/index.js"
```
I don't see `/lib/index.js` anywhere in the project directory, where is this file being called from? -- I'm interested in looking at the js itself to see what's happening and try to learn from it.

**Comment by Larzous** [comment]
@sarisia ^^ -- sorry forgot to ping you when I wrote this initially.

**Comment by sarisia** [comment, marked as answer]
Hi, thank you for asking!

We creates release commit which only contains `lib/index.js` file.
You can find these commits in [releases page](https://github.com/sarisia/actions-status-discord/releases).

![image](https://github.com/sarisia/actions-status-discord/assets/33576079/25d8deda-6e5c-4005-8cd4-102c106da4eb)

These commits are built and commited from github actions and you can find build logs here: [release · Workflow runs · sarisia/actions-status-discord](https://github.com/sarisia/actions-status-discord/actions/workflows/release.yml)

Hope this helps!
